### PR TITLE
Clean up UI for checking for duplicate emails during account creation

### LINKED ISSUE #118: Unique e-mail addresses
Make it possible to require that each user has a unique e-mail address

### PR DESCRIPTION
**Pull request by kkbrum**
Fixes #118. I made it such that emails are only considered duplicates if they want to make a user account of the same type. This hopefully addresses the concern of a student using a parent email and the parent also wanting a guardian account using the same email. I also removed the first and last names from being displayed, since that feels like a bit of a privacy issue. I also changed the default tag setting to check for duplicate emails.

### CODE REVIEW

File: esp/esp/themes/theme_data/fruitsalad/less/main.less [hunk 1]
```diff
@@ -329,8 +329,10 @@ text-transform: uppercase;
 
 #menu #tab_logo #logo {
 border: none;
-margin-left: 9px;
-margin-top: 9px;
+padding-top: 9px;
+margin-left: auto;
+margin-right: auto;
+display: block;
```
Comment (willgearty) [inline esp/esp/themes/theme_data/fruitsalad/less/main.less:335]: Why did you make these changes to the logo styling?

**Review by willgearty — COMMENTED** [review]
This looks great, thanks @kkbrum! Just one question, then it can be merged.

File: esp/esp/themes/theme_data/fruitsalad/less/main.less (hunk 1 above)
Comment (kkbrum) [reply esp/esp/themes/theme_data/fruitsalad/less/main.less:335]: oops that must be residual from working on the logo centering PR at a similar time. I'll fix that!

**Review by willgearty — APPROVED** [review]
Perfect, thank you!

**Pull request merged**